suite.go

```diff
@@ -0,0 +1,110 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
```

Comment:
I'm not entirely sure if this functionality should live in quicktest itself, or in a subpackage (e.g. github.com/frankban/quicktest/qtsuite).